src/shared/TestInfrastructure/Objects/TestGitConfiguration.cs

```diff
@@ -51,6 +53,12 @@ public bool TryGet(GitConfigurationLevel level, string name, out string value)
                     if (values.Count == 1)
                     {
                         value = values[0];
+
+                        if (type == GitConfigurationType.Path) {
```

Comment:
nit: don't use K&R style

```suggestion
                        if (type == GitConfigurationType.Path)
                        {
```